source/docs/software/wpilib-tools/wpical/index.rst

```diff
@@ -0,0 +1,177 @@
+WPIcal
+======
+.. image:: images/WPIcal.png
+    :alt: WPIcal
+
+WPIcal is a cross-platform utility that can be used to empirically measure the position and orientation of Apriltags. The primary use case for WPIcal is to make sure robots can perform as they would on an event field, but on a practice field that may have imperfect Apriltags setup.
```

Comment:
I'd recommend putting this above the picture. It's not obvious from the picture what this tool does.